compiler/rustc_lint/messages.ftl

```diff
@@ -721,6 +721,9 @@ lint_reason_must_be_string_literal = reason must be a string literal
 
 lint_reason_must_come_last = reason in lint attribute must come last
 
+lint_recursion_limit_reached =
+    reached recursion limit while {$action}
```

Comment:
This won't be translatable as it's passed a message in English as an argument. The full diagnostic should be written out here with the type as the argument.

Reply:
Thanks. Fixed.